Doc/library/functions.rst

```diff
@@ -1258,6 +1258,10 @@ are always available.  They are listed here in alphabetical order.
          * On Windows, opening a console buffer may return a subclass of
            :class:`io.RawIOBase` other than :class:`io.FileIO`.
 
+   .. versionchanged:: 3.9
+      The ``U`` mode has been removed.
```

Comment:
```suggestion
      The ``'U'`` mode has been removed.
```

Reply:
Remove the `deprecated` directive and other mentions of the `'U'` mode above.

Reply:
done and done